sdk/appcenter-identity/src/test/java/com/microsoft/appcenter/identity/IdentityTest.java

```diff
@@ -461,6 +461,16 @@ public void readCacheAndRefreshNotModified() throws Exception {
         verifyNew(PublicClientApplication.class, times(1));
     }
 
+    @Test
+    public void testSignOut() {
```

Comment:
You don't need to prefix `test` to Java tests as opposed to Objective-C tests, though we're not consistent. What about name it `signOutRemovesToken` for clarity?